City-of-Helsinki/helsinki-paatokset
# PP-214: Add council meeting live stream embed

Add live stream embed + live stream announcement block

To test:

- If you don't have Helsinki Kanava credentials set, add them to your .env.local file. Variables to add are HELSINKI_KANAVA_ID and HELSINKI_KANAVA_SECRET. You can find these in https://helsinkisolutionoffice.atlassian.net/wiki/spaces/PP/pages/2689106104/Helsinki-kanavan+videot at the bottom of the page (public repo so i won't add them here). 
- Run docker-composer up --build -d app to rebuild container
- Run printenv inside container to verify your newly added variables are loaded
- Run `make drush-cim && make drush-updb && make drush-cr`.
- Go to https://helsinki-paatokset.docker.so/fi/admin/config/helsinki-kanava/settings. Set the City council setting to point to your local node of City Council (Kaupunginvaltuusto). All meetings link should be correct from cim.
- Run `drush mim ahjo_meetings` and `drush mim paatokset_meeting_videos`.
- Meetings data contains placeholder data. For testing purposes change a meeting to have both its start date in the future and its decisions maker field value to 'Kaupunginvaltuusto'
- Change line 91 from `.../paatokset_helsinki_kanava/paatokset_helsinki_kanava.module` to always return true. Do the same for line 57 of  `.../paatokset_helsinki_kanava/src/Plugin/Block/AnnouncementsBlock`.
- Go to https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto. You should now see announcement for the next meeting as well as the live stream embed on the page.
